regparser/web/jobs/views.py

```diff
@@ -58,23 +76,24 @@ def create(self, request, *args, **kwargs):
         """
         serialized = self.get_serializer(data=request.data)
         serialized.is_valid(raise_exception=True)
-        eregs_args = [
-            "pipeline",
-            str(serialized.validated_data["cfr_title"]),
-            str(serialized.validated_data["cfr_part"]),
-            "./testing"
-        ]
+
+        eregs_args = self.build_eregs_args(serialized.validated_data)
         job = queue_eregs_job(eregs_args, timeout=60*30, result_ttl=-1)
 
         # Paranoia--validate the values we provide:
         job_id = job.id
         for validator in serialized.get_fields()["job_id"].validators:
             validator(job_id)
-        statusurl = status_url(job_id)
+        statusurl = status_url(job_id, sub_path=self.sub_path)
```

Comment:
If we're relying on this property, we probably want to emphasize it as an `abc.abstractproperty`